.babelrc

```diff
@@ -9,9 +9,6 @@
     },
     "production": {
       "plugins": ["transform-es2015-modules-commonjs"]
-    },
```

Comment:
This will be still required to support older version.

Reply:
Removing this is the whole point of this PR. [jsnext:main](https://github.com/jsforum/jsforum/issues/5) points to code with ES6 modules, not node modules. Anyone relying on this is relying on incorrect functionality.

Reply:
I understand if you think this is deserving of a major version bump, but it's incorrect as it is.

Reply:
Make sense, I guess we will need to do it as part of major bump version as this will break for the users who are using as it is now.

Reply:
Sounds like a plan, man.